include/json/value.h

```diff
@@ -344,8 +344,22 @@ Json::Value obj_value(Json::objectValue); // {}
   /// otherwise, false.
   bool empty() const;
 
-  /// Return isNull()
-  bool operator!() const;
+  /** \brief Return ! isNull();
+   *
+   * Example of usage:
+   * \code
+   * Json::Value root;
+   * std::cin >> root;
+   * 
+   * if (auto tag = root["tag"]) {
+   *    // Behavior if tag object with tag key exist
```

Comment:
This comment is probably overdoing it.
It's also got a grammatical problem and I don't like involving the behavior of operator[] into the explanation. Let's explain it without referring to other API points if we can.